### ISSUE TITLE
Data Syncing

### ISSUE THREAD

**Issue by mcrider** [arunoda/meteor-up-legacy#31]
Love meteor-up, but It would be great to have some sort of data synchronizing features. For example a 'mup sync_from_server' command to fetch the meteor DB from the remote server and load into my local, and 'mup sync_to_server' for the inverse.  When I'm working on a site locally and want load the database onto a remote server, the solution (using mongodump) is pretty arduous.

**Comment by timbrandin** [comment]
This would definitely be useful, this is a very common thing to do with Drush in Drupal.
For example doing:
`drush sql-sync @local @prod --create-db` will synchronize the database from the machine defined in the @local alias to another machine defined in the @prod alias. Very useful!

**Comment by MaazAli** [comment]
I think this should be an entirely different tool. Considering that the database of some applciations can be hosted in a location other than where your app is being deployed. Creating another tool where you can define aliases for the location of your mongo database would be pretty neat.

**Comment by timbrandin** [comment]
@MaazAli Yeah your right, this could be something else yeah.. or what do you think @arunoda? :)

**Comment by arunoda** [comment]
Yes it is. We need mup to be simple.
And that makes my life simple too. (no need to maintain a lot)

**Comment by timbrandin** [comment]
True! Would be nice if @meteor was extensible with for example aliases and data-syncing then one could do for example:
`meteor db-sync @local @prod`

**Comment by timbrandin** [comment]
Then one could also write:
`meteor mongo @prod` to access the mongo interface on the production server

**Comment by timbrandin** [comment]
And aliases could be found in `~/.meteor/aliases/.`

Which one can share the aliases with your coworkers super simple just by adding symlinking that folder to dropbox/drive etc.. :)

**Comment by timbrandin** [comment]
Actually mup could also benefit from using aliases, because we already keep deployment settings files somewhere outside of the repo most times, this could be contained in the aliases instead. and would make it possible to just stand in the right app and do mup deploy @prod

Aliases could even inherit settings from other aliases and so forth.

Sorry for spamming! :S
